src/cdk/scripts/deploy-check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+ENV_FILE=".env"
+
+# Check if .env file exists
+if [[ ! -f "$ENV_FILE" ]]; then
+    echo "Error: .env file must exist at $ENV_FILE"
+    exit 1
+fi
+
+# Load environment variables
+source "$ENV_FILE"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ShellCheck can&#39;t follow non-constant source. Use a directive to specify location.